src/modules/FSXARemoteApi.ts

```diff
@@ -131,11 +131,13 @@ export class FSXARemoteApi implements FSXAApi {
   }
 
   private getRemoteProject(remoteProject: string) {
```

Comment:
This method is no getter anymore, since we return the very same value, that we pass to the method. 
Instead we should remove the return statement and rename the method to "validateRemoteProject"